plugins/aws/resoto_plugin_aws/__init__.py

```diff
@@ -491,13 +491,36 @@ def set_name_from_org() -> bool:
                 pass
             return False
 
+        def set_name_from_profile() -> bool:
+            if account.profile:
+                account.name = account.profile
+                log.debug(f"Set name for {account.kdname} from profile")
+                return True
+            return False
+
+        # if we prefer the profile name and we have a profile
+        # we set the name from the profile and return immediately
+        if Config.aws.prefer_profile_as_account_name:
+            if Config.aws.scrape_org:
+                log.error("Setting account name from profile with scrape_org enabled is likely not what you want")
```

Comment:
All acounts in the org would have the same name?
If yes, we should ignore this setting in that case?
Logging the error will not propagate this information to the user. Consider using CoreFeedback.